external/llvm-project/mlir/unittests/Target/LLVM/SerializeROCDLTarget.cpp

```diff
@@ -114,7 +114,8 @@ TEST_F(MLIRTargetLLVMROCDL, SKIP_WITHOUT_AMDGPU(SerializeROCDLToPTX)) {
   ASSERT_TRUE(!!module);
 
   // Create a ROCDL target.
-  ROCDL::ROCDLTargetAttr target = ROCDL::ROCDLTargetAttr::get(&context);
+  ROCDL::ROCDLTargetAttr target = ROCDL::ROCDLTargetAttr::get(
+      &context, 2, "amdgcn-amd-amdhsa", "gfx900", "", "400");
```

Comment:
I'd suggest making a copy of the test, one that checks for the current default, and one that checks for 400.

Reply:
Done